.pre-commit-config.yaml

```diff
@@ -21,5 +21,4 @@ repos:
     rev: v2.7.4
     hooks:
     - id: pyupgrade
-      args: ["--py37-plus"]
```

Comment:
translation strings do not work with f-strings, so they need to be written with good old `.format(` but this is changing that, which will break localization.

Reply:
💔

Reply:
Sorry :-| 

Is there a way to conditionally skip section s in code for pyupgrade? we could do that 🤷🏼  ?

Reply:
oh I'm just being dramatic :) it's not a big deal.  definitely worth the translation upgrade.  I guess it kinda relates to the other question (about the "meaning" of translating a string that has a variable in it). right?